pkg/process/config/config.go

```diff
@@ -35,9 +35,23 @@ const (
 	defaultRuntimeCompilerOutputDir = "/var/tmp/datadog-agent/system-probe/build"
 )
 
+// Name for check performed by process-agent or system-probe
```

Comment:
I have a block comment to explain the variables since they are all exported. Let me know if it would be preferred for them to be local (only used in `config` package) or if I should do individual comment for each const.

Reply:
Block comment should be fine!